python/ray/tests/BUILD

```diff
@@ -461,6 +461,14 @@ py_test(
   data = ["//python/ray/experimental/packaging/example_pkg"],
 )
 
+py_test(
+  name = "test_runtime_env_container",
+  size = "large",
+  srcs = ["test_runtime_env_container.py"],
+  tags = ["exclusive", "team:core", "container"],
+  deps = ["//:ray_lib", ":conftest"],
+)
+
```

Comment:
nit: could you format this file properly with `buildifier`?

Reply:
I've never used `buildifier` before, how can I do that?

Reply:
I've formatted this build rule to match the indentation around it!